app-misc/lirc/lirc-0.9.4.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils systemd flag-o-matic
+
+DESCRIPTION="decode and send infra-red signals of many commonly used remote controls"
+HOMEPAGE="http://www.lirc.org/"
+
+MY_P=${PN}-${PV/_/}
+
+if [[ "${PV/_pre/}" = "${PV}" ]]; then
+	SRC_URI="mirror://sourceforge/lirc/${MY_P}.tar.bz2"
+else
+	SRC_URI="http://www.lirc.org/software/snapshots/${MY_P}.tar.bz2"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE="doc static-libs X audio irman ftdi inputlirc iguanair systemd"
+
+S="${WORKDIR}/${MY_P}"
+
+REQUIRED_USE="
+	iguanair? ( irman )
+"
+DEPEND="
+	doc? ( app-doc/doxygen )
+"
+
+RDEPEND="
+	X? (
+		x11-libs/libX11
+		x11-libs/libSM
+		x11-libs/libICE
+	)
+
+	systemd? ( sys-apps/systemd )
+
+	audio? (
+		>media-libs/portaudio-18
+		media-libs/alsa-lib
+	)
+	irman? ( media-libs/libirman )
+
+	iguanair? ( app-misc/iguanaIR )
+
+	ftdi? ( dev-embedded/libftdi:0 )
+
+	inputlirc? ( app-misc/inputlircd )
+"
+
+pkg_setup() {
+
+	# set default configure options															Any ideas what is this used for now?
+	LIRC_DRIVER_DEVICE="/dev/lirc0"
+
+	filter-flags -Wl,-O1
+}
+
+src_configure() {
+	econf \
+		$(use_enable static-libs static) \
+		$(use_with X x)
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
+
+	newinitd "${FILESDIR}"/lircd-0.8.6-r2 lircd
+	newinitd "${FILESDIR}"/lircmd lircmd
+	newconfd "${FILESDIR}"/lircd.conf.4 lircd
+
+	insinto /etc/modprobe.d/
+	newins "${FILESDIR}"/modprobed.lirc lirc.conf
+
+	newinitd "${FILESDIR}"/irexec-initd irexec
+	newconfd "${FILESDIR}"/irexec-confd irexec
+
+	if use doc ; then
+		dohtml doc/html/*.html
```

Comment:
`dohtml` is deprecated by `EAPI=6` (https://blogs.gentoo.org/mgorny/2015/11/13/the-ultimate-guide-to-eapi-6/), just `dodoc -r doc/html`